HIF.md

```diff
@@ -385,69 +386,97 @@ Only present if `HAVE_MODE_SWITCH_STAT`:
+([`REQ_DATA_TX`][tx-req]) nor received ([`IND_DATA_RX`][rx]). Some
+configuration needs to be done before calling this command, typically a PHY
+must be selected with [`SET_RADIO`][rf-set], and a unicast schedule must be
+configured with [`SET_FHSS_UC`][uc].
 
- - `bool value`  
-    Only `true` is implemented
+Body of this command is empty.
 
 ### `0x21 REQ_RADIO_LIST`
 
-Get the list of radio configuration supported by the device.
+Request the list of radio configuration supported by the device, typically
+issued by the host during a startup sequence before selecting a PHY
+configuration. The RCP will answer with a series of [`CNF_RADIO_LIST`][rf-list]
+commands.
 
 Body of this command is empty.
 
 ### `0x22 CNF_RADIO_LIST`
 
-List of radio configuration supported by the device, in response to
-`REQ_RADIO_LIST`.
+List of radio configuration supported by the RCP, as configured during project
+generation. Entries are defined in groups of mode-switch compatible PHYs. For
+OFDM configurations, all MCS are defined in the same entry.
+
+ - `uint8_t entry_size`  
+    Size of a radio configuration entry in bytes. This is meant to support API
+    evolution.
 
  - `bool list_end`  
-    If not set, the list will continue in another `CNF_RADIO_CONFIG_LIST`.
+    If not set, the list will continue in another [`CNF_RADIO_LIST`][rf-list].
 
- - `uint8_t len`  
-    Number of entries in next field.
+ - `uint8_t count`  
+    Number of radio configuration entries in next field.
 
  - `struct rf_config[]`  
     - `uint16_t flags`
         - `0x0001`: If set, this entry is in same group than the previous.
-        - `0xFFF0`: Supported MCSs (from MCS0 to MCS11)
-    - `uint8_t rail_phy_mode_id`: Same than phy_mode_id, but for OFDM, only MCS0
-      is retrieved.
-    - `uint32_t chan_base_khz`
-    - `uint32_t chan_spacing_hz`
-    - `uint16_t chan_count`
+        - `0x01FE`: Bitfield of supported OFDM MCS (from MCS0 to MCS11).
+    - `uint8_t rail_phy_mode_id`  
+       Wi-SUN PhyModeId, but for OFDM, only MCS0 exists.
+    - `uint32_t chan_f0`  
+       Frequency of the first channel in Hz.
+    - `uint32_t chan_spacing`  
+       Channel spacing in Hz.
+    - `uint16_t chan_count`  
+       Number of channels without holes in the spectrum.
+    - `uint16_t sensitivity` (API >= 2.4.0)  
+       Minimum RX sensitivity in dBm for this PHY.
 
 ### `0x23 SET_RADIO`
 
 Configure the radio parameters.
 
  - `uint8_t index`  
-    Index in the `rf_config` list
+    Index in the `rf_config` list.
 
  - `uint8_t mcs`  
-    MCS to be used if `index` points to an OFDM modulation
+    MCS to be used if `index` points to an OFDM modulation.
+
+ - `bool enable_ms` (API > 2.0.1)  
+    Enable mode switch in RX and TX, using all the PHY configurations available
+    in the same group as the selected PHY. Mode switch is enabled at PHY level
+    (PHR) and MAC level (MAC command frame). See [`REQ_DATA_TX`][tx-req] for TX
+    parameters.
```

Comment:
I thought this field only impacts Rx (Tx is managed by `REQ_DATA_TX`)

Reply:
I need to check with Alban. Code makes it not obvious.